acados/ocp_nlp/ocp_nlp_cost_ls.c

```diff
@@ -689,25 +712,24 @@ void ocp_nlp_cost_ls_update_qp_matrices(void *config_, void *dims_,
         blasfeo_dgemv_n(ny, nz, -1.0, &model->Vz,
                 0, 0, &work->tmp_nz, 0, 1.0, &work->y_ref_tilde, 0, &work->y_ref_tilde, 0);
 
-        blasfeo_dtrmm_rlnn(nu + nx, ny, 1.0, &memory->W_chol,
-                0, 0, &work->Cyt_tilde, 0, 0, &work->tmp_nv_ny,
-                           0, 0);
+        blasfeo_dtrmm_rlnn(nu + nx, ny, 1.0, &memory->W_chol, 0, 0, &work->Cyt_tilde, 0, 0, &work->tmp_nv_ny, 0, 0);
 
         // add hessian of the cost contribution
-        blasfeo_dsyrk_ln(nu + nx, ny, 1.0, &work->tmp_nv_ny, 0, 0, &work->tmp_nv_ny, 0, 0, 1.0,
+        blasfeo_dsyrk_ln(nu + nx, ny, model->scaling, &work->tmp_nv_ny, 0, 0, &work->tmp_nv_ny, 0, 0, 1.0,
```

Comment:
Isn't this changing the definition of the Lagrange term from discrete to continuous time?

Reply:
Nope, it just gives the possibility to optionally introduce a scaling. If model->scaling==1 (which is the default), nothing changes.

But scaling allows e.g. the implementation of Euler integration of continuous costs in the (continuous time) matlab interface.